### ISSUE TITLE
Add support for Python 2.6

### ISSUE THREAD

**Issue by wangst321** [nicoulaj/rainbow#19]
I use pyenv on centos6, whose default python is 2.6.6.

It seems rainbow doesn't support python2. so I install python3 use pyenv, and activate it::

```
$ pyenv install 3.4.2
$ pyenv shell 3.4.2
```

then, install rainbow::

```
$ pip install rainbow
```

run it::

```
$ rainbow --yellow='.*WARNING.*' --red='.*ERROR.*' /path/to/my.log
```

ImportError::

```
Traceback (most recent call last):
  File "/usr/bin/rainbow", line 17, in <module>
    import sys, re, os, logging, configparser
ImportError: No module named configparser
```

`configparser` is for python3, which is replacing `ConfigParser` in python2. so I check the script `/usr/bin/rainbow` , the first line is ::

```
#!/usr/bin/python
```

which is point to python2.6.6. It walks around pyenv. so I try to fix it::

```
#!/usr/bin/env python
```

It works!

**Comment by nicoulaj** [comment]
Fixed since e0836572ab912d4e4ddcfb74351697b7527a9ed0

**Comment by nicoulaj** [comment]
Python 2.6 support is broken again.
